packages/teams-js/test/public/pages.spec.ts

```diff
@@ -1299,48 +1287,6 @@ describe('Testing pages module', () => {
       }
     });
 
-    describe('Testing pages.returnFocus function', () => {
```

Comment:
This is a duplicate section. I believe we have more duplicate sections for other functions. I will make a ticket to track that these get cleaned up- for now, I'm just focusing on what would make the tests pass for the v1 merge into v2.

Reply:
Can you please double-check that are in fact straight duplicates? They might differ because we have framed and frameless unit tests.

Reply:
@shrshindeMSFT can confirm, but it does look like the second set of `returnFocus` tests are testing with frameless mock, different from the first set.

Reply:
@aeunms These are two different test cases as the communication layer goes through framed window(line 40) and frameless window(line 1302). You can check the `describe` block it says framed and frameless

Reply:
Thanks for the catch! Will un-delete.